examples/react-app/terraform/versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.71.0"
+    }
+    kubernetes = {
+      version = "~> 1.9"
+    }
```

Comment:
Use the latest version? You'll need to remove the load_config_file from the provider because that is deprecated.